main.go

```diff
@@ -0,0 +1,45 @@
+// tail | (c) 2020 NETWAYS GmbH | GPLv2+
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+)
+
+func main() {
+	buf := bufio.NewReader(os.Stdin)
+	var allData [][]byte
+	var loopCount int
+
+	for {
+		data, dataErr := buf.ReadBytes('\n')
+		if dataErr != nil && dataErr != io.EOF {
+			fmt.Fprintln(os.Stderr, dataErr)
+			os.Exit(1)
+		}
+		allData = append(allData, data)
+		if dataErr == io.EOF {
+			if bytes.Compare(data, []byte("")) == 0 {
+			}
+			break
+		}
+	}
+
+	if bytes.Compare(allData[len(allData)-1], []byte("")) == 0 {
```

Comment:
Any way to shorten `bytes.Compare(..., []byte("")) == 0`?

Reply:
i dont think so

Reply:
How long is a `[]byte("")`?

Reply:
0?

Reply:
Exactly.